src/components/Facility/EncounterCard.tsx

```diff
@@ -18,90 +29,123 @@ interface EncounterCardProps {
 
 export const EncounterCard = (props: EncounterCardProps) => {
   const { encounter } = props;
+
+  const encounterIcons = {
+    imp: BedDouble,
+    amb: Ambulance,
+    obsenc: Stethoscope,
+    emer: Building2,
+    vr: MonitorSmartphone,
+    hh: Home,
+  } as const;
```

Comment:
let's move this to constants / types; since it's potentially reusable, also do `as const satisfies Record<encounter_type, ...>`